### PR TITLE
basic test setup and fixes

### PR DESCRIPTION
**Pull request by Vovke**

### CODE REVIEW

File: src/chain/rpc.rs [hunk 1]
```diff
@@ -86,19 +95,34 @@ pub async fn get_keys_from_storage(
     storage_name: &str,
     block: &BlockHash,
 ) -> Result<Value, ErrorChain> {
+    let storage_key_prefix = format!(
+        "0x{}{}",
+        hex::encode(twox_128(prefix.as_bytes())),
+        hex::encode(twox_128(storage_name.as_bytes()))
+    );
+
+    let count = 100; // I expect it to always work, but will it actually?
```
Comment (Slesarew) [inline src/chain/rpc.rs]:
```suggestion
    let count = 100; // TODO make full scan just in case
```

File: src/chain/rpc.rs [hunk 2]
```diff
@@ -528,12 +554,12 @@ pub async fn asset_balance_at_account(
                     return Ok(Balance(value));
                 }
             }
-            Err(ErrorChain::AssetBalanceNotFound)
+            panic!();
```
Comment (Slesarew) [inline src/chain/rpc.rs]: are you reverting this to panic?

File: Cargo.toml [hunk 3]
```diff
@@ -19,6 +19,7 @@ names = { version = "0.14", default-features = false }
 
 tokio-util = { version = "0.7", features = ["rt"] }
 tokio = { version = "1", features = ["full"] }
+reqwest = { version = "0.11", features = ["json"] }
```
Comment (Slesarew) [inline Cargo.toml]: shouldn't this be dev-deps?

**Review by Slesarew — CHANGES_REQUESTED** [review]
please revert panics and unwraps to proper errors

File: src/lib.rs [hunk 4]
```diff
@@ -0,0 +1 @@
+pub mod definitions;
```
Comment (Vovke) [inline src/lib.rs:1]: to export definitions into tests

**Review by Slesarew — APPROVED** [review]

**Pull request merged**